MERCURY_ITC/iocBoot/iocMERCURY-IOC-01/st-temp-spc.cmd

```diff
@@ -4,7 +4,7 @@
 stringiftest("FLOW_SPC_PRESSURE_$(TEMP_NUM)", "$(SPC_TYPE_$(TEMP_NUM)=NONE)", 5, "FLOW")
 stringiftest("VTI_SPC_PRESSURE_$(TEMP_NUM)", "$(SPC_TYPE_$(TEMP_NUM)=NONE)", 5, "VTI")
 
-epicsEnvSet "VTI_CALIB_BASE_DIR" "$(VTI_CALIB_BASE_DIR=$(ICPCONFIGBASE)/$(INSTRUMENT))"
+epicsEnvSet "VTI_CALIB_BASE_DIR" "$(VTI_CALIB_BASE_DIR=$(ICPCONFIGBASE)/$(ICPCONFIGHOST))"
```

Comment:
are you trying to look in `C:\instrument\settings\config\<>\configurations\` ? if so i think you can remove `$(ICPCONFIGBASE)/$(ICPCONFIGHOST)` and replace with `$(ICPCONFIGROOT)` a bit like https://github.com/ISISComputingGroup/EPICS-ioc/blob/master/TC/iocBoot/iocTC-IOC-01/st-common.cmd#L7

Reply:
ignore me, im being dumb, i guess you are looking in common instead

Reply:
So the path we're actually aiming for is `C:\Instrument\Settings\config\<INST_NDX>\calib\mercury_vti_spc`

Reply:
`$(ICPCONFIGROOT)` gets you to `C:\instrument\settings\config\<>\configurations\` so you'd have to `..` - i think i prefer your current solution